tools/jenkins/jenkins.sh

```diff
@@ -45,8 +45,10 @@ build_sqoop
 
 make apps
 
-build/env/bin/pip install pylint
-ln -s $(which pylint) build/env/bin/pylint
+if ! [ -x "$(command -v pylint)" ]; then
+    build/env/bin/pip install pylint
+fi
+ln -sf $(which pylint) build/env/bin/pylint
```

Comment:
Also add it in the `if`?

Reply:
Creating a symbolic link should be outside of `if`.

`Cleaning repo.` would delete any change from the commit.
ref. https://github.com/sungpeo/hue/blob/9489b486dcbeabd2df554a87b2942892298a5171/tools/jenkins/jenkins.sh#L31-L39

It means that `build/env/bin/pylint` would be deleted. (not `SKIP_CLEAN`)